File: exercises/0x01/2-04.c
/* Condicionales
Determinar si un número es par, impar o cero */
#include <stdio.h>
int main() {
    int x;
    printf("Número: ");
    scanf("%d", &x);
    if (x == 0) {
        printf("El número es cero\n");
    }
    else if (x % 2 == 0) {
        printf("El número es par\n");
    }
    else
    {
        printf("El número es impar\n");
    }
    return 0;
}

File: exercises/0x04/2-04.py
""" Ejercicios de condicionales
Escribir un programa que pida al usuario un número entero
y muestre por pantalla si es par o impar. """
def my_code():
    n = int(input("Ingrese un número entero: "))
    if n % 2 == 0:
        print(f"{n} en un número par")
    else:
        print(f"{n} es un número impar")

my_code()